w2w/w2w.py

```diff
@@ -1148,9 +1182,14 @@ def _adjust_greenfrac_landusef(
     dst_data['LANDUSEF'] = dst_data.LANDUSEF.astype('float32')
 
     if orig_num_land_cat < 24:
```

Comment:
Will this be affected in any way?

Reply:
I can't remember really why I have this if statement there. Because of this, I suggest to leave as is?